upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -749,3 +753,113 @@ func (c *awsCloudImplementation) FindVPCInfo(vpcID string) (*fi.VPCInfo, error)
 
 	return vpcInfo, nil
 }
+
+// DefaultInstanceType determines an instance type for the specified cluster & instance group
+func (c *awsCloudImplementation) DefaultInstanceType(cluster *kops.Cluster, ig *kops.InstanceGroup) (string, error) {
+	var candidates []string
+
+	switch ig.Spec.Role {
+	case kops.InstanceGroupRoleMaster:
+		// Some regions do not (currently) support the m3 family; the c4 large is the cheapest non-burstable instance
```

Comment:
Yeah, I guess we can change with adequate notification/warnings, learning from the root volume size change.  But: we might want to see how many people that affects negatively, and I don't want to lump that in with this change.